Package.swift

```diff
@@ -42,6 +42,9 @@ extension Target {
 
 let package = Package(
   name: "RxSwift",
+  platforms: [
+    .macOS(.v10_10), .iOS(.v8), .tvOS(.v9), .watchOS(.v3)
```

Comment:
Not sure how this works but shouldn't we have Linux as well? Or is that by default?

Reply:
I thought so but I couldn't find the linux platform definition.

https://github.com/apple/swift-package-manager/blob/b060e8e46ea1d8ae15c909405bb6ff7e7251a8cb/Sources/PackageDescription4/SupportedPlatforms.swift#L28-L90